src/components/Header/header.js

```diff
@@ -0,0 +1,6 @@
+export default class Header {
+  constructor() {
+    // eslint-disable-next-line
+    console.log('header loaded');
+  }
+}
```

Comment:
This could be added to the scaffold file for js.

Reply:
yes will be afterwards, with this.log.
but i think we will wait until we have the js structure.